src/bindle_util.rs

```diff
@@ -0,0 +1,381 @@
+use std::{collections::{HashMap, HashSet}, iter::FromIterator};
+
+use bindle::{Invoice, Parcel};
+
+// TODO: this file is a bit of a cop-out but will be useful during
+// the transition.  Find better homes for these things!
```

Comment:
Is this file still to be used as-is?

Reply:
Hmm, probably.  The structure isn't great but I think the concepts are mostly right and the organisation can can be refined over time.  I'll take another look but may end up just deleting the comment.  Thanks for the catch.